### ISSUE TITLE
[Material Design] Create New BB Kit: Material Design Components

### ISSUE THREAD

**Issue by IamManchanda** [zurb/building-blocks#88]
Yes Foundation loves Flat Design but this could be gold

https://material.angular.io/components
http://www.material-ui.com/#/components/app-bar

cc / @rafibomb @kball for decision on this 

**PS**: My hands are itching to start working on it!

**Comment by IamManchanda** [comment]
Also if this is approved,
can i use Material icon library => https://material.io/icons/
instead of font awesome or foundation icons for this ??

**Comment by canterberrie** [comment]
It looks like this exists on github already: https://eucalyptuss.github.io/material-foundation/

Itt would be really cool to have a Kit for this! Could be a good opportunity to collaborate with them to make this happen.

**Comment by IamManchanda** [comment]
Yeah sure @canterberrie !

**Comment by rafibomb** [comment]
@IamManchanda Yes - make sense to use the material icon library for it. Are you using Material Design in your work?

**Comment by IamManchanda** [comment]
Unfortunately not @rafibomb
